Sources/ParseSwift/Objects Protocols/ObjectType.swift

```diff
@@ -46,6 +46,14 @@ internal extension ObjectType {
     func getEncoder() -> ParseEncoder {
         return getParseEncoder()
     }
+
+    func getTestDecoder() -> JSONDecoder {
+        return getDecoder()
+    }
+
+    func getEncoderWithoutSkippingKeys() -> ParseEncoder {
```

Comment:
This is what I meant by skipping keys, a simple change that is meant to only be used inside the framework for testing. I was checking it with `getTestDecoder`, and that may be used for other tests in the future

Reply:
This makes sense, but is there a way to have the encoders used only for testing live in the test suite itself, instead of including them in the main codebase?

Reply:
This is automatically marked internal in the framework meaning the only way it can be accessed is by using `@testable import ParseSwift`